src/map/trait.cpp

```diff
@@ -35,122 +35,32 @@
 *                                                                       *
 *                                                                       *
 ************************************************************************/
-
 CTrait::CTrait(uint8 id)
+    : m_id(id)
```

Comment:
not that it particularly matters, but if you're going to move these out of the constructor anyways, it's preferable to just have them initialized in the header

Reply:
Just want to understand your preferred standard and make sure we're on the same page.
In the past, the MIL has been faster at initializing member variables than doing it in the constructor body. I am not 100% sure that is still the case, but it's a good habit. Would you like me to just move the ctor itself into the header with an empty body?

Also, I removed the 0/null intializations. . .but I may have done that out of reflex from working in Unreal. That engine fills out ALL data with 0 at creation so I may add that back in. herp derp

Reply:
see http://en.cppreference.com/w/cpp/language/initializer_list for documenation, and https://github.com/DarkstarProject/darkstar/blob/master/src/map/instance.h#L83 for some examples

Reply:
oh. inline initializers. I can do that. Not my favorite, but you're the boss. ;)